### PR TITLE
ENH: use changedir to ensure we actually test the tarball.

### PR DESCRIPTION
**Pull request by cournape**
@sjagoe

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -3,26 +3,32 @@ envlist = py26,py27,py33,py34,pypy,flake8
...
-         coverage report --include=okonomiyaki/*
+changedir = .tox
+commands=coverage erase --rcfile=../.coveragerc
+         coverage run --rcfile=../.coveragerc -m haas
+         coverage report --rcfile=../.coveragerc --include=okonomiyaki/*
 
 [testenv:flake8]
 deps= -rdev_requirements2.txt
-commands=flake8 okonomiyaki
+commands=flake8 ../okonomiyaki
```
Comment (sjagoe) [inline tox.ini]: Does this also need a `changedir = .tox` ?

**Comment by sjagoe** [comment]
Just one question; otherwise LGTM.

File: tox.ini (hunk 1 above)
Comment (cournape) [inline tox.ini]: No, actually it was not needed in any of the `testenv:` sections.

I removed it + fixed a coverage issue

**Comment by sjagoe** [comment]
LGTM

**Pull request merged**